node/src/reactor/main_reactor/tests/transactions.rs

```diff
@@ -1549,6 +1549,7 @@ async fn refunds_are_payed_and_fees_are_on_hold_classic_pricing() {
     .await;
 }
 
+#[ignore]
```

Comment:
What's the reason for ignoring this test?

Reply:
Because it requires wasm code. My impression was, all tests that require external wasm binaries should be ignored by default. 🤓